### DISCUSSION TITLE
Make YouTube thumbnails clickable

### DISCUSSION THREAD

**Discussion by ryaneford** [glanceapp/glance#278] [General]
Hi!

Loving Glance so far, I use the YT channel list and i noticed that you can only go into the videos when you click the text on the page, and it'd be really nice to be able to click the thumbnail itself as it's more of an accessible means of getting to the video link.

**Comment by svilenmarkov** [comment]
Hey there! Thumbnails not being links is actually intended behavior due to a different reason. 

For vertical space-saving measures, videos by default are inside of a horizontal scrolling container, which on a desktop without a touchpad is far from ergonomic, however there is one thing that makes it somewhat pleasant - auto-scrolling. It's the thing you do when you press down your scroll wheel on an empty part of the page and the page starts scrolling linearly in the direction you move your cursor, going faster the further away you are from the initial position of the click. The scroll wheel button however has one more use - opening links in a new tab, which takes precedence, and that makes the two incompatible. 

If the thumbnails were links, you could technically still trigger auto-scrolling, though you'd have to do so at very specific locations, making it once again uncomfortable, where as currently you have the freedom of clicking anywhere above the video title and knowing it'll work. 

I suppose this is something I could add an option for and let people decide which behavior they prefer.
